### PR TITLE
Fix: #229 element path causes error when adding new block

### LINKED ISSUE #229: Article editor, bug when adding a block at the end after a previous block is removed
* PR target branch: `feature/articles`

# Steps to reproduce

* add 2 paragraph blocks
* delete the first paragraph
* use the `+` button on the remaining paragraph to add any block; throws an error

# Demo

https://user-images.githubusercontent.com/18054507/175791683-57c7827a-4865-43f4-9572-f845adc4e51c.mp4

# Bug probable location

https://github.com/fxhash/fxhash-website/blob/f2b4294f7ea85d9a6f74ae8fd6f7651e5d132bc3/src/components/NFTArticle/SlateEditor/Elements/RenderElements.tsx#L41

### PR DESCRIPTION
**Pull request by maerzhase**
- fix #229

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!doctype html>
```
Comment (ciphrd) [inline index.html:1]: There are some files bleeding in the push

**Review by ciphrd — CHANGES_REQUESTED** [review]
Files bleeding in commit

File: index.html (hunk 1 above)
Comment (maerzhase) [reply index.html]: sorry -.-

**Review by ciphrd — APPROVED** [review]
LGTM

**Pull request merged**